### PR TITLE
Update Github Pages Styling

### LINKED ISSUE #33: Main page: Font
Should we investigate using a specific font for the Ground website? As of right now, the site use the default font, 'Open Sans,' provided by the site theme.

Fonts can have a significant impact on readability and an audience's perception of the platform and what its use-value is.  We may wind up deciding the current font is a-ok—but should we open up a discussion?

Some font resources:

https://fonts.google.com/
https://design.google/library/choosing-web-fonts-beginners-guide/
https://thenextweb.com/dd/2017/03/31/science-behind-fonts-make-feel/

### PR DESCRIPTION
**Pull request by scolsen**
This change contains updates to the repo's doc site styling. I had to copy in the Github Pages theme's (cayman) default layout file in order to add custom fonts. We can customize this default layout in the future, if need be. Fixes #33

### CODE REVIEW

**Comment by scolsen** [comment]
Fixes #33

File: docs/assets/css/style.scss [hunk 1]
```diff
@@ -2,9 +2,42 @@
...
+  font-size: 4.5rem;
+}
+
+.project-tagline {
+  font-family: sans-serif;
+  font-weight: normal;
+  margin-bottom: 2rem;
+  font-variant: all-small-caps;
+  text-transform: none;
+}
```
Comment (gino-m) [inline docs/assets/css/style.scss]: Also should we override opacity to 0.8 here? It was kinda hard to read on the current background.

File: docs/assets/css/style.scss [hunk 2]
```diff
@@ -2,9 +2,42 @@
...
   background-repeat: no-repeat;
   background-position: center;
   background-size: cover;
 }
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: $font-heading;
+  text-transform: uppercase;
+} 
+
```
[the remark this answers is not in the document]
Comment (scolsen) [reply docs/assets/css/style.scss]: Done!

**Comment by gino-m** [comment]
Love the new look! Live now!

**Pull request merged**